packages/design-system-docs/src/pages/components/Table/Table.example.jsx

```diff
@@ -0,0 +1,56 @@
+import {
+  Table,
+  TableBody,
+  TableCaption,
+  TableDataCell,
+  TableHead,
+  TableHeaderCell,
+  TableRow,
+} from '@cmsgov/design-system';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+ReactDOM.render(
+  <Table stackBreakpoint="sm" scrollable>
```

Comment:
The stackable part is working great, but for some reason I cant get the example to show the scrollable notice. Currently the text in this example never causes the table to overflow its container and start the horizontal scrolling. This maybe can be handled in a followup, but maybe we need to update the content and styling so that theres an easy to test the scrollable behavior.

Reply:
Added another column `Links` to enable scroll behavior.